ammonite/httpMethods.sc

```diff
@@ -126,6 +126,35 @@ object Test {
                  ).get,
        keyChain = List(cosyKey,localKey)
    )
+   
+   //----------------------------------------------------------------------------------------------
+
+   def deleteLocalKey = http.run(
+     req      = DELETE[`text/turtle`](
+                  resource = Uri("https://localhost:8443/2013/")
+                )
+   )
+
+   def getLocalKey = http.run(
+      req     = GETpg(
+                  uri = Uri("https://localhost:8443/2013/"),
+                  keyChain = List(localKey)
+     )
+   )
```

Comment:
why would the key be the URL of the container? Have you tried out these functions to see what they return?